index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <h1> INTENTION TIMER </h1>
+  </header>
+  <main>
+    <section class="left-section">
+      <h2 class="new-activity" New Activity </h2>
+        <section class="left-section">
```

Comment:
Why are there two sections with the class left-section?

Reply:
Other thank the double class name here, lookin' good!